src/components/LoginCard/index.js

```diff
@@ -2,14 +2,18 @@ import './style.css';
 import logoRozetka from '../../img/logoRozetka.png';
 import Input from '../../components/Input/input';
 import Button from '../../components/Button/button';
+import { ImEye } from "react-icons/im";
+import { ImEyeBlocked } from "react-icons/im";
 
 const LoginCard = () => {
     return (
         <div id="login-card">
             <img src={logoRozetka} className="login-logo" alt="logo" />
-            <Input isLogin={true} />
-            <Input isLogin={false} />
-            <Button />
+            <Input islogin={true} />
```

Comment:
If you provide a property with "true" value, you can use this way ( <Input isLogin /> )